Gems/AWSGameLift/Scripts/gamelift_build_package_linux.py

```diff
@@ -0,0 +1,169 @@
+#
+# Copyright (c) Contributors to the Open 3D Engine Project.
+# For complete copyright and license terms please see the LICENSE at the root of this distribution.
+#
+# SPDX-License-Identifier: Apache-2.0 OR MIT
+#
+#
+
+# Python script to create the package necessary for GameLift
+# To do this by hand please see the steps provided @ https://www.o3de.org/docs/user-guide/gems/reference/aws/aws-gamelift/build-packaging-for-linux/
+
+import os 
+import shutil
+import re
+import argparse
+import itertools
+import threading
+import time
+import sys
+import stat
+
+def main():        
```

Comment:
Break main into semantic function steps so its not a giant block of code and potentially aids resuability if we had a windows version etc.